rfcs/0025-nix-core-team.md

```diff
@@ -1,11 +1,17 @@
 ---
 feature: nix-core-team
 start-date: 2018-01-31
+end-date: 2019-04-25
 author: Graham Christensen
 co-authors: Daniel Peebles, Eelco Dolstra, Peter Simons, Shea Levy, Vladimír Čunát
-related-issues:
+related-issues: #44 (disbands the team)
 ---
 
+# Superceded!
+
+This RFC is superceded by [RFC 44](./0044-disband-nix-core.md). The original text
+is preserved below.
+
```

Comment:
Nitpick: that's a repeated [typo](https://en.wiktionary.org/wiki/supercede#Verb) ;-)

Reply:
Fixed, thanks!